k2/python/csrc/torch/v2/ragged_arc.cu

```diff
@@ -58,6 +64,32 @@ RaggedArc::RaggedArc(
   // TODO: we also need to pass the name of extra_labels and ragged_labels.
 }
 
+RaggedArc::RaggedArc(const RaggedArc &src, const Ragged<Arc> &arcs,
```

Comment:
I think `from_unary_function_tensor` can be implemented as a constructor, so do `from_unary_function_ragged` and `from_binary_function_tensor`.